ed/idlpatches/trusted-types.idl.patch

```diff
@@ -0,0 +1,65 @@
+From a2a36c82e56b8956f32710c0b5d76db2e68d3544 Mon Sep 17 00:00:00 2001
+From: =?UTF-8?q?Philip=20J=C3=A4genstedt?= <philip@foolip.org>
+Date: Wed, 10 Feb 2021 13:24:47 +0100
+Subject: [PATCH] Remove trusted-types.idl overloads
+
+https://github.com/w3c/webappsec-trusted-types/issues/283
```

Comment:
Looking at the issue, it sounds like this patch should no longer be needed.

Does this PR include a mechanism to detect patches that have become redundant?

Reply:
For some reason the IDL I tried to get excluded is still there. I noticed it today but didn't look into it because it will likely take time to reach webref anyway.

If it had been removed this patch would no longer apply. Generally, the way we'll notice that something needs fixing is when patches no longer apply or when the tests start failing. I haven't written instructions for how to deal with that yet, I thought I'd learn by doing 😀